### ISSUE TITLE
Update jquery corner plugin

### ISSUE THREAD

**Issue by TimGS** [ClipperCMS/ClipperCMS#192]
Needs updating when jQuery core updated to 1.9

**Comment by microcipcip** [comment]
What is this plugin for? Looks like an hack plugin to make border radius work on Internet Explorer, as we are not supporting older versions of IE I think this should be removed.

**Comment by TimGS** [comment]
It's in WebUsers/WebLoginPE.

I don't know how it's used though.

**Comment by microcipcip** [comment]
There is also ieButtonFix.js, which again, looks like an hack for IE.

**Comment by Cipa** [comment]
Kill them all :))))

**Comment by Cipa** [comment]
jquery.corner2.js - is only used in the docs for Webusers or WebloginPE so we could keep it

ieButtonFix.js - I can't seem to get read of this one. I removed all the refereces but it is still generated in the cache file so in the frontend as well. It might be a GIt issues that; GIT might restore my file even though I manually removed it. Could someone else take care of this after I commit the WebUser update?
